docs/example/README.md

```diff
@@ -208,7 +208,7 @@ Wait for the changes to propagate and observe the counters increment for booksto
 - http://localhost:8082 - **bookstore-v2**
 
 ## Inspect Dashboards
-OSM ships with a set of pre-configured Grafana dashboards. **NOTE** If you still have the additional terminal still running the `./scripts/port-forward-all.sh` script, go ahead and `CTRL+C` to terminate the port forwarding. The `osm dashboard` port redirection will not work simultaneously with the port forwarding script still running. The `osm dashboard` can be viewed with the following command:
+OSM can be configured to deploy Grafana dashboards using the `--enable-grafana` flag in `osm install`. **NOTE** If you still have the additional terminal still running the `./scripts/port-forward-all.sh` script, go ahead and `CTRL+C` to terminate the port forwarding. The `osm dashboard` port redirection will not work simultaneously with the port forwarding script still running. The `osm dashboard` can be viewed with the following command:
```

Comment:
I was wondering whether it may be useful to mention the flags in the [Install OSM Control Plane](https://github.com/ksubrmnn/osm/blob/disable_prometheus/docs/example/README.md#install-osm-control-plane) section so that users don't reach the end of the demo and only then realize they can't use metrics if they hadn't enabled them already. Would resolve #1893 and #1897

Reply:
Addressed